.github/workflows/spanner-integration-tests-against-emulator.yaml

```
@@ -0,0 +1,52 @@
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'Spanner/**'
+  pull_request:
+    paths:
+      - 'Spanner/**'
+name: integration-tests-against-emulator
```

nit: Could we use a nice name here?
```suggestion
name: Run Spanner integration tests against service emulator
```